firmware/lm32/ci.c

```diff
@@ -128,6 +135,8 @@ static void help(void)
 	puts("");
 #endif
 	help_debug();
```

Comment:
Debug should be the last thing in the help, move the heartbeat print up a bit. Probably should go after the matrix bits?